lms/templates/courseware/syllabus.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
Includes a couple of `${get_course_syllabus_section(...)}` which seems like it might contain HTML, but not sure.  Can you verify?

Reply:
Fixed